addons/mrp/views/mrp_bom_views.xml

```diff
@@ -170,7 +170,7 @@
                         </page>
                     </notebook>
                     </sheet>
-                    <chatter/>
+                    <chatter reload_on_attachment="True"/>
```

Comment:
```suggestion
                    <chatter reload_on_attachment="True open_attachments="True""/>
```
The display looks a bit jittery when we add an attachment, as it reloads itself and the opened attachment section is immediately closed. I think (and checked with mgm about that) that it would make sense to always display the attachments instead for the following reasons :
- Aside from a little flicker from the reload, at least we still see the document we just added
- It makes sense to know on a bom that a document is linked (since it's not likely to be a huge amount of documents)